src/scenes/home/team/team.js

```diff
@@ -1,58 +1,93 @@
-          Operation Code deeply appreciates the time, energy, and hard work of
-          our <b>Founding Board Members</b>: Aimee Knight, Laura Gomez, Pete
-          Runyon, Josh Carter, and Nick Frost.
-        </p>
-
-        <p style={{ textAlign: 'center' }}>
-          <em>Thank you for setting us up for success!</em>
-        </p>
-      </div>
-    </Section>
-
-    <Section title="Our Executive Staff" theme="white">
-      <div className={styles.executiveStaff}>
-        {ExecutiveStaff.map(staff => (
-          <TeamCard
-            key={`${staff.name} + ${staff.role}`}
-            name={staff.name}
-            role={staff.role}
-            slack={staff.slackUsername}
-            email={staff.email}
-            isBoard={false}
-          />
-        ))}
+class Team extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      teamMembers: null,
+      errorResponse: false
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://api.operationcode.org/api/v1/team_members')
+      .then(response => this.setState({ teamMembers: response.data }))
+      .catch(() => this.setState({ errorResponse: true }));
+  }
+
+  render() {
+    return (
+      <div>
+        <QuoteBanner
+          author="Abraham Lincoln"
+          quote="To care for him who shall have borne the battle and for his widow, and his orphan."
+        />
+
+        {this.state.errorResponse && (
+          <p className={styles.teamError}>
+            Whoops! Something went wrong… please check back later.
+          </p>
+        )}
+
+        {this.state.teamMembers && (
+          <Section title="Our Board" theme="white">
+            <div className={styles.boardMembers}>
+              {this.state.teamMembers
+                .filter(board => board.group === 'board')
+                .map(members => (
+                  <TeamCard
+                    key={`${members.name} + ${members.role}`}
+                    name={members.name}
+                    role={members.role}
+                    description={members.description}
+                    imageSrc={members.image_src}
+                  />
+                ))
+              }
+            </div>
+
+            <div className={styles.foundingMembers}>
+              <p>
+                Operation Code deeply appreciates the time, energy, and hard work of
+                our <b>Founding Board Members</b>: Aimee Knight, Laura Gomez, Pete
+                Runyon, Josh Carter, and Nick Frost.
+              </p>
+              <p style={{ textAlign: 'center' }}>
+                <em>Thank you for setting us up for success!</em>
+              </p>
+            </div>
+          </Section>
+        )}
+
+        {this.state.teamMembers && (
+          <Section title="Our Executive Staff" theme="white">
+            <div className={styles.executiveStaff}>
+              {this.state.teamMembers
+                .filter(executive => executive.group !== 'board')
```

Comment:
In [this backend comment](https://github.com/OperationCode/operationcode_backend/issues/230#issuecomment-366469813) there's a link to current state of the `TeamMember` db in prod.

You'll want to confirm the accuracy of that data before committing to pulling the executive staff from the db.

cc: @kylemh @ksmacleod99

Reply:
While I was testing the TeamMember Collection endpoint I noticed that only the board members 
have a group. I assume that the other members are the executive staff. Am I wrong?

Reply:
While I was testing the TeamMember collection endpoint I noticed that only the board members have a group. You could do the same for the executive staff. I will change the code accordingly.